README.md

```diff
@@ -1,5 +1,3 @@
-# W3C WebDriver Specification
```

Comment:
It looked quite odd with just a sentence under each heading, but I can add them all back in if you'd like.